### ISSUE TITLE
xmobar fails to compile (with ghc 9.0.2?)

### ISSUE THREAD

**Issue by Hypoon** [gentoo-haskell/gentoo-haskell#1256]
It looks like this is the problem:
```
setup: Encountered missing or private dependencies:
base >=4.11.0 && <4.15
```

**Comment by hololeap** [comment]
Thanks for the report. I'll try to get this sorted out for ya.

**Comment by hololeap** [comment]
I assume you are trying to compile version `0.37`

This should fix it: https://github.com/gentoo-haskell/gentoo-haskell/commit/c2638763d2f05728430685fbddfd05b2cf70c83d

Sync and try again

**Comment by solpeth** [comment]
Closing, as both the fixed version 0.37 and version 0.41 should build on ghc-9.0.2 based on local testing. Please reopen if any issues.

**Comment by Hypoon** [comment]
Thanks for the quick fix! I can confirm that it resolved the issue.
